HWMgmt/OCPBaselineRedfishService.json

```diff
@@ -0,0 +1,571 @@
+                },
+                "Message": {
+                    "ReadRequirement": "Recommended"
+                },
+                "MessageId": {},
+                "MesageArgs": {
+                    "ReadRequirement": "Recommended"
+                },
+                "Severity": {}
+            }
+        },
+        "LogService": {
+            "PropertyRequirements": {
+                "DateTime": {},
+                "DateTimeLocalOffset": {},
+                "Entries": {},
+                "LogEntryType": {},
+                "LogPurposes": {
+                    "ReadRequirement": "Recommended"
+                },
+                "ServiceEnabled": {
+                    "ReadRequirement": "Recommended"
+                }
+            },
+            "ActionRequirements": {
+                "ClearLog": {
+                    "ReadRequirement": "Recommended"
+                }
+            }
+        },
+        "Manager": {
+            "PropertyRequirements": {
+                "DateTime": {
+                    "ReadRequirement": "Mandatory",
+                    "WriteRequirement": "Recommended"
+                },
+                "DateTimeOffset": {
+                    "ReadRequirement": "Mandatory",
+                    "WriteRequirement": "Recommended"
+                },
+                "ManagerType": {},
+                "UUID": {},
+                "Status": {
+                    "PropertyRequirements": {
+                        "Health": {},
+                        "State": {}
+                    }
+                },
+                "FirmwareVersion": {},
+                "NetworkProtocol": {
+                    "ReadRequirement": "Supported"
+                },
+                "EthernetInterfaces": {},
+                "LogServices": {},
+                "Links": {
+                    "PropertyRequirements": {
+                        "ManagerForChassis": {
+                            "ReadRequirement": "Supported"
+                        }
+                    }
+                }
+            },
+            "ActionRequirements": {
+                "Reset": {
+                    "Parameters": {
+                        "ResetType": {
+                            "ParameterValues": [
+                                "ForceRestart"
+                            ]
+                        }
+                    }
+                }
+            }
+        },
+        "ManagerAccount": {
+            "PropertyRequirements": {
+                "AccountTypes": {},
+                "Enabled": {},
+                "Links": {
+                    "PropertyRequirements": {
+                        "Role": {}
+                    }
+                },
+                "Password": {},
+                "PasswordChangeRequired": {},
+                "PasswordExpiration": {},
+                "RoleId": {},
+                "UserName": {}
+            }
+        },
+        "ManagerNetworkProtocol": {
```

Comment:
Any write requirements in here? I think we need to make SSDP mandatory per language in the spec regarding the need to let a user disable it.

Reply:
Agreed - while SSDP is marked as recommended, if it's supported, at least `ProtocolEnabled` must be RW.